<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Material</class>
 <widget class="QDialog" name="Material">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create/Modify Material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Material #                                    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="matNum"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Bulk Modulus                               </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="KLine">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Poission's Ratio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="KCombo">
         <item>
          <property name="text">
           <string>Constant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Function of Effective Vertical Stress</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bulkValLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="vLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="kLabel">
         <property name="text">
          <string>Vertical hydraulic conductivity kv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="kLine">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Ratio kh/kv</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="kCombo">
         <item>
          <property name="text">
           <string>Constant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Function of Effective Vertical Stress</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="kValLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ratioLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Unit weight                                  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Initial Void Ratio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>ks/kh factor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="densLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="voidRatioLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="CdLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add/Modify Material</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setCd">
       <property name="text">
        <string>Set Cd to All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setRatio">
       <property name="text">
        <string>Set Ratio to All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listInfor">
       <property name="text">
        <string>List Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
